src/tfr_control/src/controller.cpp

```diff
@@ -35,6 +35,16 @@ namespace tfr_control
 
     void Controller::write() {
         // TODO: Waiting on further hardware development to implement
+        if (use_fake_values) {
+            // Update all actuators velocity with the command (effort in).
+            // Then update the position as derivative of the velocity over time.
+            for (int i = 0; i < 7; i++) {
```

Comment:
What is this 7 based on?

Reply:
The 7 is based on the number of actuators we have registered with the controller_manager. If we were to change the number of sensors/actuators, this number would also have to change. As this number is set in stone by design, I'm not worried about that here. There may be an incoming issue with the implementation of the hardware itself later but that shouldn't affect this (it's only on the hardware side, not with registering with controller_manager), and that's reserved for the second hardware interface card.

Reply:
Wouldn't it be better to use a variable instead? It is something simple that could easily be forgetten